.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,27 @@
+name: Feature or Enhancement Request
+description: Request a new feature or enhancement
+labels: ["enhancement"]
+title: "[FEAT]: "
```

I've been leaning towards no abbreviations

```suggestion
title: "[FEATURE]: "
```